process/transaction/shardProcess.go

```diff
@@ -708,9 +708,6 @@ func (txProc *txProcessor) getUserTxCost(
 	userTxHash []byte,
 	destTxType process.TransactionType,
 ) *big.Int {
-	if !txProc.flagPenalizedTooMuchGas.IsSet() {
```

Comment:
Not need this for backwards compatibility?

Reply:
no - this is used in relayed tx only